proto/options.proto

```diff
@@ -309,6 +309,9 @@ message Costing {
     }
     uint32 fixed_speed = 80;
     uint32 axle_count = 81;
+    oneof has_use_lit {
+      float use_lit = 82;
+    }
```

Comment:
if the default is 0.f then you dont need to use `oneof` because both not being set and the default are one and the same value. i know that all the other ones, even the ones that default to 0 are using `oneof` but thats only because i didnt get a chance to go back and do these, you could be the guy to start the wave :D

Reply:
went the way of a new macro and no `oneof`